hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/SimpleConfiguration.java

```diff
@@ -93,4 +98,12 @@ public int getPort() {
   public long getWaitTime() {
     return waitTime;
   }
+
+  public Class<? extends Object> getMyClass() {
```

Comment:
Nit: this can be simplified to:

```suggestion
  public Class<?> getMyClass() {
```